scripts/py_matter_yamltests/matter_yamltests/parser.py

```diff
@@ -34,9 +34,12 @@
     'command',
     'disabled',
     'endpoint',
+    'event',
```

Comment:
While I get why you are adding event and eventNumber here, we don't process these properly. How do you feel about still throwing an error if these are provided? Because I don't want anyone to get a false sense that this is working.

If you do want to keep this without erroring can you please populate a field in TestStep so that the content is accessible

Reply:
I am fine to keep them out for now. `TestEvents.yaml` parsing is still broken with or without this fix anyway.

Reply:
I mean I would assume that the test using `groupId` would also fail since the value is not used in any meaningful way currently. Up to you the direction you'd like to go

Reply:
I have removed them.